code/game/objects/items/RPD.dm

```diff
@@ -479,6 +481,15 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 /obj/item/pipe_dispenser/proc/activate()
 	playsound(get_turf(src), 'sound/items/deconstruct.ogg', 50, TRUE)
 
+/obj/item/pipe_dispenser/proc/mouse_wheeled(datum/source, mob/user, delta_x, delta_y, params)
+	SIGNAL_HANDLER
+
+	if(delta_y > 0)
+		piping_layer = min(PIPING_LAYER_MAX, piping_layer + 1)
+	else
```

Comment:
If the person side-scrolls, this will cause the layer to go down as if they had scrolled down.

Reply:
I don't think I have ever even seen a mouse that you can scroll sideways with, but I guess I could change the else to an else if.

Reply:
They exist. They're not even uncommon. I have a cheapy $20 wireless mouse that does, and my gaming mouse does. It's usually a clicky button on the side or clicking the scroll wheel itself side-to-side.

Reply:
well, I changed it to an `else if`